packages/kbn-scout/src/playwright/config/index.ts

```diff
@@ -27,10 +27,11 @@ export function createPlaywrightConfig(options: ScoutPlaywrightOptions): Playwri
     reporter: [
       ['html', { outputFolder: './output/reports', open: 'never' }], // HTML report configuration
       ['json', { outputFile: './output/reports/test-results.json' }], // JSON report
+      scoutPlaywrightReporter(), // Scout report
     ],
     /* Shared settings for all the projects below. See https://playwright.dev/docs/api/class-testoptions. */
     use: {
-      serversConfigDir: Path.resolve(REPO_ROOT, '.scout', 'servers'),
+      serversConfigDir: SCOUT_SERVERS_ROOT,
```

Comment:
I'm trying to understand the reason to have '@kbn/scout-info' package. The paths it exports are technically reporting-related, right? 
Question: what is the benefit of not putting it and exporting from `kbn/scout-reporting` ?

Reply:
`@kbn/test` and `@kbn/scout` depend on `@kbn/scout-reporting` to build and/or use a reporter. So, yes. We could live without the `@kbn/scout-info` package.

But, `SCOUT_SERVERS_ROOT` is _technically_ not reporting-related and I expect we'll start to identify more constants similar to it in the future.

I'm just being wary of a conflict in the future where we could end up having a circular dependency between `@kbn/scout` and `@kbn/scout-reporting`. The info package can serve as a bridge for any info that we might want to share between those two.

I also think it's a good idea to have this package available where we keep well-defined information used to operate Scout, rather than burden to the user with identifying conventions on their own and re-compose that info if they need it.